SH4RE/SH4RE/Messages/MessagesInboxView.swift

```diff
@@ -67,20 +70,23 @@ struct MessagesInboxView: View {
     }
     
     private var messagesView: some View {
+        
         ScrollView {
                 ForEach(vm.recentMessages) { recentMessage in
+                    
                     VStack {
                         Button {
                             let uid = FirebaseManager.shared.auth.currentUser?.uid == recentMessage.fromId ? recentMessage.toId : recentMessage.fromId
                             
                             self.chatUser = .init(id: uid, uid: uid, name: recentMessage.name)
-                            
                             self.chatLogViewModel.chatUser = self.chatUser
                             self.chatLogViewModel.fetchMessages()
+                            self.chatLogViewModel.profilePic = profilePicDict[recentMessage.toId] ?? UIImage(named: "ProfilePhotoPlaceholder")!
                             self.shouldNavigateToChatLogView.toggle()
+                            
                         } label: {
                             HStack(spacing: 16) {
-                                Image("ProfilePhotoPlaceholder") // TODO: this will become a profile picture
+                                Image(uiImage: profilePicDict[recentMessage.toId] ?? UIImage(named: "ProfilePhotoPlaceholder")!) // TODO: this will become a profile picture
```

Comment:
can remove TODO comment

Reply:
We must remind ourselves of our past successes